apps/website/docs/react/guides/atomic-css.md

```diff
@@ -4,4 +4,104 @@ sidebar_position: 1
 
 # Atomic CSS
 
-This page is not ready yet.
+Atomic CSS is an opposite approach to monolithic classes. In Atomic CSS every property-value is written as a single CSS rule.
+
+```html
+<!-- Monolithic classes -->
+<button class="button"></button>
+<!-- Atomic CSS -->
+<button class="display-flex align-items"></button>
+```
+
+```css
+/* Monolithic classes */
+.button {
+  display: flex;
+  align-items: center;
+}
+
+/* Atomic CSS */
+.display-flex {
+  display: flex;
+}
+.align-items-center {
+  align-items: center;
+}
+```
+
+Atomic CSS enables CSS rules re-use and reduces total amount of defined rules. Other components can re-use the same CSS rules and create fewer rules as styles grow.
+
+## Style overrides
+
+While Griffel works by default in runtime it has [ahead-of-time compilation](/react/ahead-of-time-compilation/introduction). AOT compilation process [cannot create new rules at runtime](/react/guides/limitations) and because of that we rely on [merging CSS rules](/react/api/merge-classes).
```

Comment:
```suggestion
While Griffel works by default in runtime it has [ahead-of-time compilation](/react/ahead-of-time-compilation/introduction). The AOT compilation process [cannot create new rules at runtime](/react/guides/limitations) and because of that, we rely on [merging CSS rules](/react/api/merge-classes) to apply conditional styles at runtime.
```